DiscountDiscover/Managers/LocationManager.m

```diff
@@ -23,6 +23,8 @@ - (id)init {
     }
     
     [self.locationManager startUpdatingLocation];
```

Comment:
At this point the location can be nil, outdated, or not accurate, so it's not really useful. What you would usually do is to trigger location updates earlier in the app lifecycle, and only save it when the delegate is notified with a valid and accurate-enough value.